opentelemetry-kotlin-compat/src/jvmTest/resources/span_attrs.json

```diff
@@ -20,22 +20,22 @@
     },
     "startTimestampNs": 0,
     "attributes": {
-      "long_key": "42",
-      "long_list_key": "[42]",
-      "bool_key": "true",
-      "double_list_key": "[3.14]",
-      "string_list_key": "[a]",
-      "bool_list_key": "[true]",
-      "double_key": "3.14",
-      "string_key": "second_value"
+      "bool_key":"true",
+      "bool_list_key":"[true]",
+      "double_key":"3.14",
+      "double_list_key":"[3.14]",
+      "long_key":"42",
+      "long_list_key":"[42]",
+      "string_key":"second_value",
+      "string_list_key":"[a]"
     },
     "events": [],
     "links": [],
     "endTimestampNs": 0,
     "ended": true,
     "totalRecordedEvents": 0,
     "totalRecordedLinks": 0,
-    "totalAttributeCount": 9,
+    "totalAttributeCount": 8,
```

Comment:
We add one attribute with the same key twice. In OTel Java this was allowed, in OTel Kotlin it's currently not.

Reply:
https://opentelemetry.io/docs/specs/otel/common/#attribute-collections

>The keys in each such collection are unique, i.e. there MUST NOT exist more than one key-value pair with the same key.

It looks like this could potentially be a bug in the Java SDK implementation.